# Translation of Odoo Server.
# This file contains the translation of the following modules:
# 	* account_ponto
# 
# Translators:
# 敬雲 林 <chingyun@yuanchih-consult.com>, 2020
# 
msgid ""
msgstr ""
"Project-Id-Version: Odoo Server saas~13.5+e\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-09-01 07:39+0000\n"
"PO-Revision-Date: 2020-09-07 08:18+0000\n"
"Last-Translator: 敬雲 林 <chingyun@yuanchih-consult.com>, 2020\n"
"Language-Team: Chinese (Taiwan) (https://www.transifex.com/odoo/teams/41243/zh_TW/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: zh_TW\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid ""
"Access to ponto using token is being deprecated. Please follow migration "
"process on "
"https://docs.google.com/document/d/1apzAtCgZl5mfEz5-Z8iETqd6WXGbV0R2KuAvEL87rBI"
msgstr ""

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "Account"
msgstr "帳戶"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "An error has occurred: %s"
msgstr ""

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "An error occurred during online synchronization"
msgstr "線上同步期間發生錯誤"

#. module: account_ponto
#: model:ir.model.fields,field_description:account_ponto.field_account_online_journal__display_name
#: model:ir.model.fields,field_description:account_ponto.field_account_online_provider__display_name
#: model:ir.model.fields,field_description:account_ponto.field_account_online_wizard__display_name
msgid "Display Name"
msgstr "顯示名稱"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "Fetching transactions took too much time."
msgstr ""

#. module: account_ponto
#: model:ir.model.fields,field_description:account_ponto.field_account_online_journal__id
#: model:ir.model.fields,field_description:account_ponto.field_account_online_provider__id
#: model:ir.model.fields,field_description:account_ponto.field_account_online_wizard__id
msgid "ID"
msgstr "ID"

#. module: account_ponto
#: model:ir.model,name:account_ponto.model_account_online_journal
msgid "Interface for Online Account Journal"
msgstr "線上會計帳簿接口"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "Invalid access keys"
msgstr ""

#. module: account_ponto
#: model:ir.model.fields,field_description:account_ponto.field_account_online_journal____last_update
#: model:ir.model.fields,field_description:account_ponto.field_account_online_provider____last_update
#: model:ir.model.fields,field_description:account_ponto.field_account_online_wizard____last_update
msgid "Last Modified on"
msgstr "最後修改於"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "Link your Ponto account"
msgstr ""

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#: model:ir.model.fields.selection,name:account_ponto.selection__account_online_provider__provider_type__ponto
#, python-format
msgid "Ponto"
msgstr ""

#. module: account_ponto
#: model:ir.model.fields,field_description:account_ponto.field_account_online_journal__ponto_last_synchronization_identifier
msgid "Ponto Last Synchronization Identifier"
msgstr ""

#. module: account_ponto
#: model:ir.model.fields,field_description:account_ponto.field_account_online_provider__ponto_token
msgid "Ponto Token"
msgstr ""

#. module: account_ponto
#: model:ir.model.fields,field_description:account_ponto.field_account_online_provider__provider_type
msgid "Provider Type"
msgstr "提供商類型"

#. module: account_ponto
#: model:ir.model,name:account_ponto.model_account_online_provider
msgid "Provider for online account synchronization"
msgstr "線上帳戶同步的提供商"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "Server not reachable, please try again later"
msgstr "伺服器無法訪問，請稍後再試"

#. module: account_ponto
#: model:ir.model.fields,help:account_ponto.field_account_online_provider__ponto_token
msgid "Technical field that contains the ponto token"
msgstr ""

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "The following error happened during the synchronization: %s"
msgstr "在同步期間發生以下錯誤：%s"

#. module: account_ponto
#: code:addons/account_ponto/models/ponto.py:0
#, python-format
msgid "Timeout: the server did not reply within 60s"
msgstr "超時：伺服器沒在60秒內響應"

#. module: account_ponto
#: model_terms:ir.ui.view,arch_db:account_ponto.ponto_online_provider_account_form_view
msgid "Update Accounts"
msgstr ""

#. module: account_ponto
#: model:ir.model,name:account_ponto.model_account_online_wizard
msgid "Wizard to link synchronized accounts to journal"
msgstr "勾稽同步帳戶到日記帳嚮導"

#. module: account_ponto
#: model:ir.model.fields,help:account_ponto.field_account_online_journal__ponto_last_synchronization_identifier
msgid "id of ponto synchronization"
msgstr ""
